File: src/main/java/no/nav/fo/veilarbregistrering/arbeidsforhold/resources/ArbeidsforholdApi.java
package no.nav.fo.veilarbregistrering.arbeidsforhold.resources;

import io.swagger.v3.oas.annotations.tags.Tag;
import io.swagger.v3.oas.annotations.Operation;

@Tag(name = "ArbeidsforholdResource")
public interface ArbeidsforholdApi {

    @Operation(summary = "Henter informasjon om brukers siste arbeidsforhold.")
    ArbeidsforholdDto hentSisteArbeidsforhold();
}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/arbeidsforhold/resources/ArbeidsforholdApi.kt
package no.nav.fo.veilarbregistrering.arbeidsforhold.resources

import io.swagger.v3.oas.annotations.Operation
import io.swagger.v3.oas.annotations.tags.Tag

@Tag(name = "ArbeidsforholdResource")
interface ArbeidsforholdApi {
    @Operation(summary = "Henter informasjon om brukers siste arbeidsforhold.")
    fun hentSisteArbeidsforhold(): ArbeidsforholdDto?
}